ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -649,7 +420,9 @@ class Tab: NSObject {
       url = newURL
       notifyObservers = true
     } else {
-      if isTabVisible(), url?.displayURL != nil, url?.displayURL?.scheme == "about", !loading {
+      if tabDelegate?.isTabVisible(self) == true, url?.displayURL != nil,
+        url?.displayURL?.scheme == "about", !loading
```

Comment:
`url?.displayURL?.scheme == "about"` will also catch `url?.displayURL != nil`

Reply:
Good catch, this code is going away in the next PR anyways, so I'll make sure its updated there